sql/builtin_functions.cc

```diff
@@ -778,17 +778,64 @@ registerMod(mod, std::make_shared<IntegerValueInfo>(), "mod");
 double ln(double v)
 {
     if (v <= 0)
-        throw HttpReturnException(400, "the argument of the ln function must"
-                                       " be strictly positive");
+        cerr << "negative value encountered in ln" << endl;
```

Comment:
Please remove the cerr, this could happen on every row and kill performance in a very nasty way.